pyscriptjs/Makefile

```diff
@@ -89,12 +89,11 @@ test:
 	make test-ts
 	make test-py
 	make test-integration-parallel
-	make test-examples
 
 # run all integration tests *including examples* sequentially
```

Comment:
here the comment should be changed:
`including examples` --> `excluding examples`